tslib/src/functions-throttled/ps-report-2-pupil-data/index.ts

```diff
@@ -0,0 +1,21 @@
+import { AzureFunction, Context } from '@azure/functions'
+import { performance } from 'perf_hooks'
+import { PsReportService } from './ps-report.service'
+import { PupilResult } from './models'
+
+const functionName = 'ps-report-2-pupil-data'
+
+const serviceBusQueueTrigger: AzureFunction = async function (context: Context, incomingMessage: any): Promise<void> {
```

Comment:
could you type `incomingMessage` to give more clarity on the message structure?

Reply:
Yes, I intend to.